native_client/BUILD

```diff
@@ -52,6 +52,8 @@ tf_cc_shared_object(
     linkopts = select({
         "//tensorflow:darwin": [],
         "//conditions:default": [
+            "-ldl",
```

Comment:
Wouldn't we need it as well on macOS ?

Reply:
Works fine on my machine, and it passes tests. I don't know why :)